x/quarantine/spec/05_queries.md

```diff
@@ -0,0 +1,76 @@
+<!--
+order: 5
+-->
+
+# gRPC Queries
+
+## Query/IsQuarantined
+
+To find out if an account is quarantined, use `QueryIsQuarantinedRequest`.
+The query takes in a `to_address` and outputs `true` if the address is quarantined, or `false` otherwise.
+
+Request:
+
++++ https://github.com/provenance-io/cosmos-sdk/blob/prov/dwedul/1046-bank-quarantine/proto/cosmos/quarantine/v1beta1/query.proto#L46-50
```

Comment:
This currently points to `prov/dwedul/1046-bank-quarantine` branch. They'll break after merge when the branch is deleted. Do we update this after merge to point to main?

Reply:
Yeah. We'll need to update them. They'll be okay as long as the branch isn't deleted (even after getting merged).

I'll make a follow-up ticket for fixing them though.